pkg/trace/schema/mempool.go

```diff
@@ -115,8 +116,18 @@ func WriteMempoolPeerState(client *trace.Client, peer p2p.ID, stateUpdate, trans
 }
 
 const (
-// LocalTable is the tracing "measurement" (aka table) for the local mempool
-// updates, such as when a tx is added or removed.
-// TODO: actually implement the local mempool tracing
-// LocalTable = "mempool_local"
+	MempoolRejectedTable = "mempool_rejected"
+	ReasonFieldKey       = "reason"
 )
+
+// WriteMempoolRejected records why a transaction was rejected.
+func WriteMempoolRejected(client *trace.Client, reason string) {
```

Comment:
Isn't it necessary to trace evicted transactions in InfluxDB, similar to how they are being traced in Prometheus?

Reply:
I believe we have another metric that already does that, so this metric was only added to complement that one.